src/Facebook/FacebookClient.php

```diff
@@ -197,7 +197,7 @@ public function prepareRequestMessage(FacebookRequest $request)
      */
     public function sendRequest(FacebookRequest $request)
     {
-        if (get_class($request) === 'FacebookRequest') {
+        if (get_class($request) === 'Facebook\FacebookRequest') {
```

Comment:
Because this:

```php
if (!$request instanceof FacebookBatchRequest) {
```

Explains why access token validation gets skipped a lot better than this:

```php
if (get_class($request) === 'Facebook\FacebookRequest') {
```

It also prevents more hard-coded fully-qualified class names. :)